cluster/config-defaults.yaml

```diff
@@ -230,6 +230,8 @@ skipper_time_based_scaling_target: "1"
 # ClusterScalingSchedules
 # hyped-article-releases
 skipper_scaling_schedule_hyped_articles_target: ""
+# custom schedules in form of "<name1>=<target1>,<name2>=<target2>"
+skipper_scaling_schedule_custom: ""
```

Comment:
`name` should be distinct, right?
Otherwise we will create a likely failing metrics service and maybe impair all HPA of the cluster

Reply:
Name is the schedule name

Reply:
and schedule name is unique per cluster

Reply:
I have updated the comment a bit. It would be interesting to test how HPA behaves when configured with duplicate object metrics; from the description
> If you provide multiple such metric blocks, the HorizontalPodAutoscaler will consider each metric in turn. The HorizontalPodAutoscaler will calculate proposed replica counts for each metric, and then choose the one with the highest replica count.

I would say it would not be different as using a single metric.

Reply:
@AlexanderYastrebov Yeah, I think you're right it would be the same as a single metric, the details are in our implementation that's why I'm not 100% sure without testing, but I think you're right.